tests/integration/test_image_filters.py

```diff
@@ -612,6 +612,37 @@ def test_rag_threshold_filter_on_rgb_image(pil_image, expected_image):
     )
 
 
+def _create_rag_mask(pil_image):
+    mask = np.ones((pil_image.size[1], pil_image.size[0]))
+    mask[:100, :] = 0
+    mask[:, :100] = 0
+    mask[-100:, :] = 0
+    mask[:, -100:] = 0
+    return mask
+
+
+@pytest.mark.parametrize(
+    "pil_image, expected_image, mask",
+    (
+        (
+            RGB.DIAGNOSTIC_SLIDE_THUMB_RGB,
+            "mask-arrays/diagnostic-slide-thumb-rgb-rag-threshold-labels",
+            None,
+        ),
+        (
+            RGB.DIAGNOSTIC_SLIDE_THUMB_RGB,
+            "mask-arrays/diagnostic-slide-thumb-rgb-rag-threshold-maskedlabels",
+            _create_rag_mask(RGB.DIAGNOSTIC_SLIDE_THUMB_RGB),
+        ),
+    ),
+)
+def test_rag_threshold_filter_return_labels(pil_image, expected_image, mask):
+    expected_value = load_expectation(expected_image, type_="npy")
+
+    ragged = imf.rag_threshold(pil_image, 650, 20.6, 9, return_labels=True, mask=mask)
```

Comment:
```suggestion
    ragged = imf.rag_threshold(pil_image, 650, 20.6, 9, return_labels=True, mask=mask)
    
```

we have a convention: between the fixtures definition, the function/method call and the assertions we have a blank line :)